tests/unit/spread-shellcheck/task.yaml

```diff
@@ -0,0 +1,19 @@
+summary: Verify spread task scripts with shellcheck
+
+# need a recent shellcheck version
+systems: [ubuntu-18.04-64]
+
+prepare: |
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB/pkgdb.sh"
+    distro_install_package python-pip shellcheck
```

Comment:
If this test and the static checks are run on the same machine, depending on ordering, shellcheck will be run on all shell files or not.

I'd rather they were checked always, independently of ordering. Could you also distro-install shellcheck on 18.04 before running the static checks?

Reply:
spellcheck in 16.04 won't work for us though, afair